src/kili/use_cases/project/project.py

```diff
@@ -28,18 +28,61 @@ def create_project(
         json_interface: Dict,
         title: str,
         description: str,
+        project_id: Optional[ProjectId],
         project_type: Optional[ProjectType],
         compliance_tags: Optional[ListOrTuple[ComplianceTag]],
     ) -> ProjectId:
-        """Create a project."""
-        project_id = self._kili_api_gateway.create_project(
-            input_type=input_type,
-            json_interface=json_interface,
-            title=title,
-            description=description,
-            project_type=project_type,
-            compliance_tags=compliance_tags,
-        )
+        """Create or copy a project if project_id is set."""
+        if project_id is not None:
+            project_copied = self._kili_api_gateway.get_project(
+                project_id=project_id, fields=["jsonInterface", "instructions"]
+            )
+            project_tag = self._kili_api_gateway.list_tags_by_project(
+                project_id=project_id, fields=["id"]
+            )
+            new_project_id = self._kili_api_gateway.create_project(
+                input_type=input_type,
+                json_interface=project_copied["jsonInterface"],
+                title=title,
+                description=description,
+                project_type=project_type,
+                compliance_tags=compliance_tags,
+            )
+            if project_copied["instructions"]:
+                self.update_properties_in_project(
+                    project_id=new_project_id,
+                    can_navigate_between_assets=None,
+                    can_skip_asset=None,
+                    compliance_tags=None,
+                    consensus_mark=None,
+                    consensus_tot_coverage=None,
+                    description=None,
+                    honeypot_mark=None,
+                    instructions=project_copied["instructions"],
+                    input_type=None,
+                    json_interface=None,
+                    min_consensus_size=None,
+                    number_of_assets=None,
+                    number_of_skipped_assets=None,
+                    number_of_remaining_assets=None,
+                    number_of_reviewed_assets=None,
+                    review_coverage=None,
+                    should_relaunch_kpi_computation=None,
+                    use_honeypot=None,
+                    metadata_types=None,
+                    title=None,
+                )
+            for tag in project_tag:
+                self._kili_api_gateway.check_tag(project_id=new_project_id, tag_id=tag["id"])
```

Comment:
Tags exist from an organization point of view.

If user of organization A is admin of project P of organization B, and try to create a project from P, it may have an error as tags will differ between organizations

Reply:
Ok I have update the code for use the method from the `TagUseCases` which already handle this case and throw an error if it's the case

Reply:
so maybe check before if we are on the same organization of the project or if tags exists on your organization.
If it's not the case, we should warn the user that tags hasnt been copied

Reply:
Ok I have update the code accordingly